<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Manevr_Dialog</class>
 <widget class="QDialog" name="Manevr_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>679</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Редагування маневру</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,8,0">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string/>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Назва маршруту</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="route_name_edit">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Радагування маневру</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,0">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QGroupBox" name="groupBox_3">
            <property name="title">
             <string>Данні маневру</string>
            </property>
            <layout class="QFormLayout" name="formLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Назва маневру</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="name_edit"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Час входу</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QDateEdit" name="dateEdit">
               <property name="enabled">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QTimeEdit" name="timeEdit">
               <property name="enabled">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Швидкість м/с</string>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>ХТТ входу</string>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="QComboBox" name="comboBox"/>
             </item>
             <item row="8" column="0">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>ХТТ виходу</string>
               </property>
              </widget>
             </item>
             <item row="8" column="1">
              <widget class="QComboBox" name="comboBox_2"/>
             </item>
             <item row="10" column="0">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Тип маневру</string>
               </property>
              </widget>
             </item>
             <item row="10" column="1">
              <widget class="QComboBox" name="comboBox_3"/>
             </item>
             <item row="12" column="0">
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>Число повторів</string>
               </property>
              </widget>
             </item>
             <item row="12" column="1">
              <widget class="QLineEdit" name="number_repeat_edit"/>
             </item>
             <item row="3" column="1">
              <widget class="QLineEdit" name="speed_edit"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_4">
            <property name="title">
             <string>Операції  с ХТТ</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>Вибір ХТТ</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QPushButton" name="point_add_button">
               <property name="text">
                <string>Додати</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0" colspan="3">
              <widget class="QComboBox" name="comboBox_4"/>
             </item>
             <item row="2" column="1">
              <widget class="QPushButton" name="point_edit_button">
               <property name="text">
                <string>Редагувати</string>
               </property>
              </widget>
             </item>
             <item row="2" column="2">
              <widget class="QPushButton" name="point_delete_button">
               <property name="text">
                <string>Видалити</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>ХТТ маневру</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Координати</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="geo_coord_radioB">
            <property name="text">
             <string>географічні</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="local_coord_radioB">
            <property name="text">
             <string>локальні</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableWidget" name="tableWidget"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QDialogButtonBox" name="buttonBox">
         <property name="standardButtons">
          <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
